kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -683,6 +683,21 @@ define([
         return jobState;
     }
 
+    /**
+     * createJobStatusSummary
+     * This is designed for use in the appCell's `execMessage` element.
+     *
+     * @param {object} jobState object representing the current job state
+     *
+     * @returns {string} a single line summary of the current state
+     */
+    function createJobStatusSummary(jobState) {
+        if (jobState && jobState.status === JOB.does_not_exist) {
+            return 'Job not found';
+        }
```

Comment:
basically just a big shortcut for adding the message `Job not found` to the cell title bar instead of a great long blurb.